components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/nats/NatsTask.java

```diff
@@ -0,0 +1,49 @@
+package org.wso2.carbon.inbound.endpoint.protocol.nats;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.core.SynapseEnvironment;
+import org.wso2.carbon.inbound.endpoint.common.InboundTask;
+
+import java.util.Properties;
+
+public class NatsTask extends InboundTask {
+
+    private static final Log log = LogFactory.getLog(NatsTask.class.getName());
+    private NatsPollingConsumer natsPollingConsumer;
+
+    NatsTask(NatsPollingConsumer natsPollingConsumer, long interval) {
+        printDebugLog("Initializing.");
```

Comment:
Initialising what? Better to mention the name of the inbound endpoint so that the user knows what NATS EP is initialising when there are multiple.

Reply:
Same applies for following DEBUG logs as well.